releasenotes/notes/java-crash-tracking-b7b240529ec26798.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Automatically enable crash tracking for Java applications
```

Comment:
The changelog is meant for customers. Alone, this sentence does not make a lot of sense. It's more something like:
```
Auto-instrumentation admission controller now automatically activates crash tracking for Java applications
```

Also this change targets the Cluster Agent, so the `releasenote` should be created in `releasenote-dca` with `reno -d releasenotes-dca`

Reply:
Agreeing with @vboulineau 's comment.

Reply:
I changed the release note wording and move it to the proposed location